HelpSource/Classes/TGrains.schelp

```diff
@@ -3,12 +3,43 @@ summary:: Buffer granulator.
 categories::  UGens>Buffer, UGens>Generators>Granular
 
 Description::
+
+/////////////////////
+/////////////////////
+/////////////////////
+
 Triggers generate grains from a buffer. Each grain has a Hanning envelope
 code::
 (sin2(x) for x from 0 to π)
 ::
 and is panned between two channels of multiple outputs.
 
+/////////////////////
+/////////////////////
+/////////////////////
+
+one cool thing you can do, is actually grabbing really large buffers (i.e.: one hour length buffers), and loading them into a buffer, and scrubbing through it in all possible forms. it works particularly well with post-modern kinds of musicm but also with another genres of music, and is a typical kind of scenario you might want to use when you grab a controller such as a wiimote, or a bi-dimensional kind of thing
```

Comment:
kislelk, i tried to fix this. i would appreciate if you could give me any further feedback on my corrections. let's move forward towards attempting to make my second contribution merging with the main supercollider repository, and who knows being part of a future release